ConditionsService/data/conditions_01.txt

```diff
@@ -64,7 +64,7 @@ double crv.digitizationPeriod = 12.55; //ns
 
 //CRV calibration parameters 
 //This is only a placeholder. In the future, these values need to be a function of time and SiPM number.
-double crv.pedestal           = 100;    //ADC
+double crv.pedestal           = 0;      //ADC
```

Comment:
Please talk to Ray about designing a calibration condition to describe this pedestal.